src/libs/hooks/usePostFavorite.ts

```diff
@@ -1,15 +1,21 @@
-import { useMutation } from 'react-query';
+import { useMutation, useQueryClient } from 'react-query';
 import { useNavigate } from 'react-router-dom';
 
 import postFavorite from '../api/postFavorite';
 
 const usePostFavorite = () => {
   const navigate = useNavigate();
+  const queryClient = useQueryClient();
   const mutation = useMutation({
     mutationFn: (bookId: number) => {
       return postFavorite(bookId);
     },
     onError: () => navigate('/error'),
+    onSuccess: () => {
+      queryClient.refetchQueries(['get-my-lecueBook'], {
+        exact: true,
+      });
+    },
   });
   return mutation;
```

Comment:
P2: useMutation은 mutate를 위해 존재하는 함수라서 mutate를 반환하는게 함수 역할에 더 맞는 것 같다는 생각이 드네용!
mutate까지 반환하면 요 훅을 가져다 쓰는 컴포넌트에서도 매번 mutate를 부를 필요가 없어서 코드도 더 깔끔해질 것 같아요!
혹시 가능하다면 mutation 함수들 아래처럼 변경해보는건 어떨까요!?
```suggestion
  return mutation.mutate;
```

Reply:
마이페이지 말구 PostFavorite에서도 mutation 쓰이는데, 요게 home에서도 쓰이더라구요..! home에도 같이 반영해두도록 할게요!

Reply:
오 !!!!!!!! 이거 저도 배워감니다